tendermint/src/time.rs

```diff
@@ -36,13 +38,38 @@ impl Time {
         Ok(Time(DateTime::parse_from_rfc3339(s)?.with_timezone(&Utc)))
     }
 
+    /// Returns an RFC 3339 string, such as 1996-12-19T16:39:57-08:00.
+    pub fn to_rfc3339(&self) -> String {
+        self.0.to_rfc3339()
```

Comment:
If you want to support this, it should be Z-normalized, which I think it is but that shiuld be reflected in the comment.